app/helpers/svg-jar.js

```diff
@@ -1,9 +1,17 @@
 import makeHelper from 'ember-svg-jar/utils/make-helper';
 import makeSVG from 'ember-svg-jar/utils/make-svg';
-import inlineAssets from '../inline-assets';
+
+function loader(assetId) {
+  try {
+    /* eslint-disable global-require */
+    return require(`ember-svg-jar/inlined/${assetId}`).default;
```

Comment:
This way we will have multiple HTTP requests, that can slow down the first app load. What do you think?

Reply:
This doesn't trigger an HTTP request, it's just loading a module that was already embedded in vendor.js.

(Implementing the symbol strategy for multiple addons on the other hand *might* cause multiple HTTP requests, which is one reason I didn't get to that yet.)